src/app.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = (src, dest) => {
+  if (src === dest || !src || !dest) {
+    return;
```

Comment:
Here you shouldn't just return from function. Instead of it you need to throw an Error
```suggestion
    throw new Error('Unable to copy file');
```